crates/service/src/index/mod.rs

```diff
@@ -69,6 +69,7 @@ pub struct IndexStat {
     pub growing: Vec<u32>,
     pub write: u32,
     pub options: IndexOptions,
+    pub size: u64,
```

Comment:
It'll be better if we return info of all segments (id, type, length, size) directly.